core/workspace/src/Project.ts

```diff
@@ -18,6 +18,7 @@ class ClassMap extends Map<Constructor<any>, any> {
 }
 
 export class Project {
+  readonly isProject = true;
```

Comment:
From what I saw this variable is not used anywhere should we had it to the class Project?

Reply:
good point! removed